### ISSUE TITLE
Bitmap fails with offset messages

### ISSUE THREAD

**Issue by ilezcano** [Thomas-Tsai/partclone#93]
Hi. When I try to create an image for some BTRFS partitions, I get a million messages as follows:

btrfsclone.c: offset(134957510656) larger than device size(53686043136), skip it.
btrfsclone.c: offset(134960365568) larger than device size(53686043136), skip it.
btrfsclone.c: offset(134954635264) larger than device size(53686043136), skip it.
btrfsclone.c: offset(134958198784) larger than device size(53686043136), skip it.
btrfsclone.c: offset(134957887488) larger than device size(53686043136), skip it.

btrfs-check runs but doesn't seem to make a difference.

**Comment by Thomas-Tsai** [comment]
hello,
could you make sure the partition in one disk? or btrfs volume in one physical partitoin?
currently, we are not support raid btrfs, did you enabled it?

Thanks.

**Comment by ilezcano** [comment]
No, this is not a multiple device or partition filesystem. However, it's not device 1 in the filesystem. It's devid 3. There's no devid 1 or 2, just 3. It got that way because in the distant past I migrated the filesystem off of a spindle onto this SSD, and adding and deleting partitions was my method for moving it.

**Comment by ilezcano** [comment]
Any ideas?
